projects/charger/inc/charger_can.h

```diff
@@ -38,6 +38,9 @@ typedef struct ChargerCanTxDataImpl {
   uint16_t max_voltage;
   uint16_t max_current;
   uint8_t charging;
+  uint8_t reserved_1;
```

Comment:
Is this just padding?

Reply:
Yeah, it is defined in the spec but it is labelled as `RESERVED` so the DLC needs to be 8 bytes even though only 5 bytes appear to be useful. I just want to follow the spec here just to be sure.